linera-core/src/unit_tests/client_tests.rs

```diff
@@ -2432,6 +2432,18 @@ where
     let client2 = builder.add_initial_chain(description2, Amount::ONE).await?;
     let client3 = builder.add_initial_chain(description3, Amount::ONE).await?;
 
+    // Use fast blocks by default on all chains.
```

Comment:
`ChangeOwnership` makes it a fast round?

Reply:
I change the ownership so that the client is now a _super_ owner of the chain. Super owners propose fast blocks by default. (But, with this change, they will switch to regular blocks if blobs are read or other oracles are used.)

Reply:
Ah, can you put bit more details into the comment then :/ I forgot about this.